### DISCUSSION TITLE
Search for my repositories

### DISCUSSION THREAD

**Discussion by elijahjcobb** [community/community#5561] [Mobile]
Search within the mobile app for just MY repos.

**Comment by aconsuegra** [comment]
Thanks for your feature request @elijahjcobb! I'm going to pass it along and we'll think about how to better include this feature in the apps.

**Reply by egvijayanand** [reply]
Also plan to include a filter to segregate Public/Private repos to locate the item quickly.

Need to scroll through the items every single time, painful and time consuming.

**Reply by terrisgit** [reply]
One year later.. we need this! Thank you

**Reply by aconsuegra** [reply]
Hi, we recently added support for Search & Filtering in repositories.

**Reply by cquanu** [reply]
Thanks @aconsuegra. It will be good to also have personal and organisation repository filter
